### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Akilankm** [Telefonica/prometheus-kafka-adapter#83]
can someone upload a proper usage documentation ?

**Comment by palmerabollo** [comment]
@Akilankm the main README https://github.com/Telefonica/prometheus-kafka-adapter#prometheus-kafka-adapter should be enough to start using it. What do you miss or what parts are not clear enough? Thanks.

**Comment by Akilankm** [comment]
- We have configured as mentioned in README and we are having the adapter in kubenetes cluster with an external prometheus and configured the nodeport as well. 

 - There seems to be a error in deployment.yaml file regarding Liveness and Readiness since the pods keeps going into **CRASHLOOPBACKOFF** state. To resolve this we commented out the lines to get it to running state

 - We are not sure how to run the adapter (we ran using `kubectl port-forward <podname>`) with remote write at prometheus end. We are getting a `socat error:connection refused` from adapter end 
 `read failed connection closed by peer` from prometheus end.

**Can this be addressed in the documentation ?**
